PluginSystem/PluginEngine.py

```diff
@@ -8,14 +8,16 @@
 """
 Plugin Engine
 On init the engine will load all the modules inside of the plugins
-directory and add them to the list of available classes stored in PluginBase.
+directory and add them to the list of available classes stored in PluginManager.
 """
 class PluginEngine:
+    historicalCallbacks = {}
+    realtimeCallbacks = {}
```

Comment:
probably not necessary to have both. historical engine and realtime engine would ideally be entirely different classes, so no need to have both of these in one. we probably won't get to historical stuff at all, so just having one plain PluginEngine with just pluginCallbacks is probably best